products/zoningtaxlots/bash/export.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+source bash/config.sh
+set_error_traps
+
+echo "Generate csvs and shapefiles ..."
+rm -rf output && mkdir -p output
+(
+    cd output
+
+    cp ../source_data_versions.csv ./
+    cp ../build_metadata.json ./
+    
+    csv_export zoningtaxlots zoningtaxlot_db &
+
+    csv_export qa_freq qaqc_frequency &
+
+    csv_export qa_bbl qaqc_bbl &
+
+    csv_export qa_mismatch qaqc_mismatch &
+
+    csv_export qa_freq qaqc_bbl &
+
+    csv_export qa_freq qaqc_bbl &
+
+    csv_export qa_vers_comparison qc_versioncomparison &
+
+    csv_export qa_null qaqc_null &
+
+    run_sql_command "\copy (
```

Comment:
Seems like this can also be a `csv_export` - if there's a difference between this and the table (column is dropped, etc), let's modify the sql so that a simple `csv_export` gives us the right output